reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -7,16 +7,15 @@ ms.date: 08/06/2018
 # Using Visual Studio Code for PowerShell Development
 
 In addition to the [PowerShell ISE][ise], PowerShell is also well-supported in Visual Studio Code
-(VSCode). Furthermore, the ISE is not supported with PowerShell Core, while VSCode is
-supported for PowerShell Core on all platforms (Windows, macOS, and Linux)
+(VSCode). The ISE isn't supported with PowerShell Core, but VSCode is supported for PowerShell Core
+on all platforms: Windows, macOS, and Linux.
 
-You can use VSCode on Windows with PowerShell version 5 by using Windows 10 or by
-installing
+You can use VSCode on Windows with PowerShell version 5 by using Windows 10 or by installing
 [Windows Management Framework 5.0 RTM](https://devblogs.microsoft.com/powershell/windows-management-framework-wmf-5-0-rtm-is-now-available-via-the-microsoft-update-catalog/)
```

Comment:
I don't think WMF 5.0 is available for download anymore. We need to direct people to install WMF 5.1. See https://docs.microsoft.com/powershell/scripting/wmf/setup/install-configure. We should link to this article for install instructions.